### PR TITLE
Add column mappers for char and URI

### PR DESCRIPTION
**Pull request by magnusr**
Fixes issue #176

### CODE REVIEW

**Comment by stevenschlansker** [comment]
Awesome!

**Comment by qualidafial** [comment]
:clap:

File: src/main/java/org/skife/jdbi/v2/util/CharColumnMapper.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+    @Override
+    public Character mapColumn(ResultSet r, String columnLabel, StatementContext ctx) throws SQLException {
+        return charFromString(r.getString(columnLabel));
+    }
+
+    public Character charFromString(String s) {
+        if (s != null) {
+            return s.charAt(0);
+        }
```
Comment (qualidafial) [inline src/main/java/org/skife/jdbi/v2/util/CharColumnMapper.java]: Hmm, this will throw an error for empty string.

Comment (magnusr) [inline src/main/java/org/skife/jdbi/v2/util/CharColumnMapper.java]: Right, yes. Will fix.

Comment (stevenschlansker) [inline src/main/java/org/skife/jdbi/v2/util/CharColumnMapper.java]: Whoops!  sorry I missed that...

Comment (stevenschlansker) [inline src/main/java/org/skife/jdbi/v2/util/CharColumnMapper.java]: Fixed in https://github.com/jdbi/jdbi/commit/cc98a82ff789aec032fb247f4fdb521b79f8b5d1

Comment (magnusr) [inline src/main/java/org/skife/jdbi/v2/util/CharColumnMapper.java]: Thanks!

**Pull request merged**